.pipelines/pr.yml

```diff
@@ -9,7 +9,10 @@ pr:
       - docs/*
 
 pool:
-  vmImage: ubuntu-latest
+  vmImage: ubuntu-18.04
```

Comment:
What is the reason for not using the latest?

Reply:
I wanted to specify the image version instead of using the latest, which could be changed in the future. But I doubt it would affect our test result. I can change it back to latest